choices.js

```diff
@@ -318,14 +318,19 @@ module.exports = {
 		{ id: 'DecodeLoop', label: 'DecodeLoop' },
 	],
 
+	TALLY_MODE: [
+		{ id: '1', label: 'Tally On' },
+		{ id: '0', label: 'Tally Off' },
+	],
+
 	// ############
 	// NDI ENCODE Look Ups
 	// ############
 
 	ENCODE_TXPM: [
-		{ id: 'Mulitcast', label: 'Mulitcast' },
+		{ id: 'Multicast', label: 'Multicast' },
 		{ id: 'TCP', label: 'TCP' },
-		{ id: 'Multi-TCP', label: 'Multi-TCP' },
+		{ id: 'MultiTCP', label: 'Multi-TCP' },
```

Comment:
I set it as Multi-TCP, as that's what the API returns..

Reply:
Ah good point. I reverted / changed some others to make sure they match as well.